charts/alfresco-connector-ms365/templates/deployment-connector-ms365.yaml

```diff
@@ -62,17 +62,23 @@ spec:
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
           {{- include "component-security-context" .Values | indent 8 }}
-          envFrom:
-            - configMapRef:
-                name: {{ .Values.repository.existingConfigMap | default (print (include "alfresco-connector-ms365.fullname" .)) }}
+           {{- $repo_details_cm := .Values.repository.existingConfigMap.name | default (print (include "alfresco-connector-ms365.fullname" .)) }}
           env:
-            {{- range $key, $value := (omit .Values.environment "JAVA_OPTS") }}
-            - name: {{ $key }}
-              value: {{ $value }}
-            {{- end }}
```

Comment:
I don't think you should remove that. You still want to have all other options as per the `.environment` values specified. Only JAVA_OPTS are a special case and treated separately to make sure we include the baseUrl option